peercrawler.py

```diff
@@ -478,6 +478,28 @@ def main():
     if args.beta: ctx = betactx
     if args.test: ctx = testctx
 
+    # setup logging
+    file_name = "network.log"
+    max_lines = 10000
+
+    # trim log file to stay under max_lines
```

Comment:
Is this only trimming lines at startup?
Will a long running session create huge files?

Reply:
Yes, it only trims at startup. I don't think this is a huge issue though, even a file with hundreds of thousands of lines would be just a few megabytes.
If you think it's worth it, I can make something that trims the file even when the program is running.

Reply:
What might end up being an issue is the size of the file being too big to be regularly served by the web app. I will change that.

Reply:
In my view, our logging needs at the moment are simple and we do not need a logging library.
We just need a class or function to store the last 10 thousands lines and make them available via a URL.
It should just be just a handful of lines of very simple code that requires no other expertise to understand and no extra documentation to read.

Reply:
Creating our own logging code would be way more complicated than what I did in this PR. The usage of the module is as simple as using logging.information("message") or other methods for different levels but with the same signature.
The logging module is also included with Python by default, so no extra dependencies are needed.